Dockerfile

```diff
@@ -18,7 +18,8 @@ WORKDIR /app/
 COPY pyproject.toml poetry.lock  ./
 RUN poetry install -n -E sentry --no-root --only=main
 COPY hathor ./hathor
-COPY README.md ./
+# BUILD_VERSION is optional to exist
+COPY README.md BUILD_VERSIO[N] ./
```

Comment:
- Does the stage-0 need this for any thing? The next stage could copy BUILD_VERSION directly, right?
- Also the `[N]` is just to force the completer to not complete and allow it to be optional, right?

Reply:
Yes, the next stage could copy directly.

I can remove this from stage-0.

The `[N]` is just to allow it to be optional, otherwise the build would fail if the file doesn't exist.

Reply:
I fixed it.